helm/cluster-cloud-director/templates/kubeadmcontrolplane.yaml

```diff
@@ -13,6 +13,11 @@ spec:
         - localhost
         - 127.0.0.1
         - "api.{{ include "resource.default.name" $ }}.{{ .Values.baseDomain }}"
+        {{- if .Values.apiServer.certSANs }}
+        {{- range .Values.apiServer.certSANs}}
+        - {{ . }}
+        {{- end -}}
```

Comment:
Are you sure it shouldn't be `{{- end }}`?

Reply:
Templating worked fine but changed to `{{- end }}` to match other uses of it.